ui/src/app/app-routing.tsx

```diff
@@ -98,14 +98,15 @@ export const AppRoutingComponent: React.FunctionComponent<RoutingProps> = ({onSi
           }} />}
       />
       <AppRoute path='/nih-callback' component={() => <HomepagePage routeData={{title: 'Homepage'}}/>} />
-    </ProtectedRoutes>
 
-    <ProtectedRoutes guards={[signInGuard, registrationGuard]}>
-      <AppRoute
+      <ProtectedRoutes guards={[signInGuard, registrationGuard]}>
```

Comment:
Shouldn't `signInGuard` not be necessary on this line, since it's nested under the other protected routes component?